cpu_scheduling_algorithms/non_preemptive_sjf_scheduling.cpp

```diff
@@ -0,0 +1,306 @@
+/**
+ * @file
+ * @brief Implementation of SJF CPU scheduling algorithm
+ * @details
+ * shortest job first (SJF), also known as shortest job next (SJN), is a scheduling policy 
+ * that selects for execution the waiting process with the smallest execution time.
+ * SJN is a non-preemptive algorithm. Shortest remaining time is a preemptive variant of SJN.
+ * @link https://www.guru99.com/shortest-job-first-sjf-scheduling.html
+ * @author [Lakshmi Srikumar](https://github.com/LakshmiSrikumar)
+ */
+
+#include <algorithm>         /// for sorting
+#include <cassert>          /// for assert
```

Comment:
lint your file using clang-tidy/clang-format

Reply:
I have done the clang format now . @realstealthninja Kindly review the PR.